grabbit/src/main/proto/node.proto

```diff
@@ -9,6 +9,7 @@ option java_outer_classname = "NodeProtos";
 message Node {
     required string name = 1;
     required Properties properties = 2;
+    repeated Node mandatoryChildNode = 3;
```

Comment:
I think `mandatoryChildNode` is correct here due to repeated qualifier (see `repeated Property property` below).  Otherwise our generated methods will be get, set, addMandatoryChildNodes(...) plural when in reality they only handle one node at a time.